src/web/components/Card/Card.tsx

```diff
@@ -178,9 +179,25 @@ export const Card = ({
                                                     }
                                                 />
                                             </>
-                                        )}
-                                    </>
-                                </CardFooter>
+                                        ) : (
+                                            undefined
+                                        )
+                                    }
+                                    commentCount={
+                                        longCount && shortCount ? (
+                                            <>
+                                                <CardCommentCount
+                                                    designType={designType}
+                                                    pillar={pillar}
+                                                    long={longCount}
+                                                    short={shortCount}
+                                                />
+                                            </>
+                                        ) : (
+                                            undefined
```

Comment:
if you dnt need the condition then you wont need this

Reply:
Ah, but `Type '"" | Element' is not assignable to type 'Element | undefined'`